core/config/entity.go

```diff
@@ -9,6 +10,7 @@ type Entity struct {
 	Version string
 
 	Sentinel SentinelConfig
+	props map[string]string
```

Comment:
It's better to have some unit test to verify that props were deserialized correctly.

Reply:
accept